stacks/spica/src/function/pages/subscription-index/subscription-index.component.html

```diff
@@ -25,8 +25,8 @@ <h4 class="mat-h4">
       </ng-container>
 
       <ng-container matColumnDef="actions">
-        <mat-header-cell *matHeaderCellDef>Actions</mat-header-cell>
-        <mat-cell *matCellDef="let element">
+        <mat-header-cell class="action-cell" *matHeaderCellDef>Actions</mat-header-cell>
+        <mat-cell class="action-cell" *matCellDef="let element">
```

Comment:
Consider `mat-header-cell.action` selector

Reply:
This class is not my custom class. This class using by Angular Material.